providers/gov/nih/normalizer.py

```diff
@@ -71,7 +71,9 @@ def format_org_address(self, doc):
 
 
 class AgentRelation(Parser):
-    related = Delegate(AwardeeAgent, ctx)
+    schema = 'AgentWorkRelation'
```

Comment:
[NIH](https://projectreporter.nih.gov/project_info_details.cfm?aid=9152186&icde=32214317) labels it as an 'Awardee organization' but I don't know what that relationship would be called. I would think the grant applicant would be the PI though.